crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/controller/ContributionControllerCalculateContributionIntegrationTest.java

```diff
@@ -59,12 +57,13 @@ class ContributionControllerCalculateContributionIntegrationTest {
     @Autowired
     private WebApplicationContext webApplicationContext;
 
-    @AfterEach
-    void after() {
-        wiremock.resetAll();
+    @BeforeEach
+    void setup() throws JsonProcessingException {
+        stubForOAuth();
```

Comment:
Whats the reason for moving the stubbing for oauth into the `@BeforeEach` hook, seems a bit inefficient to set this up again before every test, or do the stubs get cleared down after every test due to the new `@EnableWiremock` annotation?

Reply:
`@InjectWireMock` injects into static fields only, because the underlying test framework treats the WireMock server as a shared static resource to ensure consistent port binding across tests. Unfortunately, `@BeforeAll` methods can’t access static-injected fields unless they're also static.

However, I have managed to make some slight improvements and moved the filter configuration to `@BeforeAll` and removed a redundant `ObjectMapper` instantiation in the OAuth stub setup.